appserver/monitoring-console/core/src/main/java/fish/payara/monitoring/admin/SetMonitoringConsoleConfigurationCommand.java

```diff
@@ -95,18 +95,23 @@
     @Param(optional = true)
     private Boolean enabled;
 
+    @SuppressWarnings("squid:S116")
```

Comment:
NB. This is so sonar will no more complain about the `_` in the name which makes those hidden parameters.